### PR TITLE
migrate mmonit to config.schema

### PR DESCRIPTION
**Pull request by LindsayHill**
### Status 

- pack extension, complete

### Description

Migrated to `config.schema.yaml`

### Checklist (tick everything that applies)

- [X] Metadata: pack.yaml, icon, structure (required for new packs)
- [X] Version bump (required for changed packs)
- [X] Code linting (required, can be done after the PR checks)

### CODE REVIEW

File: packs/mmonit/config.schema.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+      username:
+        description: "Username to connect to mmonit for sensor"
+        type: "string"
+        required: true
+      password:
+        description: "Password to connect to mmonit for sensor"
+        type: "string"
+        required: true
+      event_types:
+        description: "Comma separated list of events to listen for. Possible values: 0=success, 1=error, 2=change. 1 and 2 by default"
```
Comment (Kami) [inline packs/mmonit/config.schema.yaml]: Ideally this should be an array and the code should convert it to that format but that's a backward incompatible change we can do some time later.

**Comment by Kami** [comment]
Merged, thanks.

File: packs/mmonit/config.schema.yaml (hunk 1 above)
Comment (Kami) [inline packs/mmonit/config.schema.yaml]: I decided to do it anyway in #632.

**Comment by LindsayHill** [comment]
Thanks!

**Pull request merged**